Core/src/main/java/com/theprogrammingturkey/comz/game/actions/DoorSetupAction.java

```diff
@@ -83,17 +83,35 @@ public void onPlayerInteractEvent(PlayerInteractEvent event)
 		if(door.arePointsFinal())
 			return;
 
-		if(event.getAction().equals(Action.LEFT_CLICK_AIR) || event.getAction().equals(Action.LEFT_CLICK_BLOCK))
+		// if(event.getAction().equals(Action.LEFT_CLICK_AIR) || event.getAction().equals(Action.LEFT_CLICK_BLOCK))
```

Comment:
No need to comment this, just go ahead and delete it

Reply:
It just need to be removed

Reply:
Done